### ISSUE TITLE
Learning Rate Scheduler 파라미터 조정 제안

### ISSUE THREAD

**Issue by EJueon** [boostcampaitech4nlp2/level1_semantictextsimilarity_nlp-level1-nlp-10#22]
**목적**
Learning Rate Scheduler 파라미터 조정을 통한 성능 개선

**설명**
https://github.com/boostcampaitech4nlp2/level1_semantictextsimilarity_nlp-level1-nlp-10/blob/66a47d536af6432269c0340ea6b90ecf1afddfb0/code/sts/model.py#L59

현재 코드에서는 T_max와 eta_min이 임의로 설정된 상태입니다. 각 파라미터의 의미는 다음을 참고하시면 될 것 같습니다. 
https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.CosineAnnealingLR.html

T_max 파라미터에 따라 다음 learning rate의 증가 및 감소 주기가 변경될 수 있으며 
<img width="808" alt="image" src="https://user-images.githubusercontent.com/93572176/198934965-9c4e369d-d10a-48ec-8ab5-a61781a43d92.png">

eta_min 수치에 따라 최소 learning_rate에 대한 제약을 걸 수 있어 성능에 영향을 미칠 수 있을듯 합니다. 

관련하여 논의 후에 조정해봐도 좋을 것 같습니다.
